sig-ml-ops/charter.md

```diff
@@ -0,0 +1,53 @@
+# Charter
+
+This charter adheres to the conventions described in the [Open Data Hub Charter Guide](../sig-charter-guide.md) and uses the Roles and Organization Management outlined in [sig-goverance](../sig-governance.md).
+
+## Scope
+
+To create services for deploying and managing machine learning models and pipelines in OpenShift and across the hybrid cloud. Our users are defined as data scientists who are interested in incorporating ML Ops capabilities into their workflows.
+
+### In scope
+
+* Services and APIs for pipelines and model serving ecosystem
+* Define and advocate for the ML Ops focused data scientist persona within the larger ODH community
+* Ensure any platform and backend changes directly benefit the user
+* Offer ease of use solutions through ODH website/tutorials
+* Assist the ML Developer Experience SIG in developing ML Ops centric ODH Dashboard extensions
+
+### Code, Images, Design, and Services Covered
+
+* ODH Pipelines Operator
```

Comment:
Is "ODH Pipelines Operator" still valid?

Reply:
Possibly. We've discussed an operator existing to abstract away some pipelines specifics , but we've not made a decision on going one way or the other.